ang/contactlayout/contactlayout.js

```diff
@@ -1,4 +1,4 @@
-(function(angular, $, _) {
+(function(angular, $, _, RELATIONSHIP_TYPES) {
```

Comment:
Rather than injecting this into the closure please just set it inside the closure (e.g. like on L47). That way we don't mess up Civi's internal closure optimization.